spec/helper.js

```diff
@@ -43,9 +43,9 @@ var defaultConfiguration = {
 };
 
 // Set up a default API server for testing with default configuration.
-var api = new ParseServer(defaultConfiguration);
+global.parseServerObject = new ParseServer(defaultConfiguration);
```

Comment:
Yeah, this is a side effect of the pull/update/push promise behavior that happens in the constructor.

Adjusted so that setServerConfiguration returns the constructed parse server object instead of using a new global, is that a fair compromise?

The tests are split into two groups: testing ParseServer constructor behavior & testing the settings router e2e.  Testing the constructor behavior benefits from having the constructed Parse Server object.